test/createRoutineCreator.js

```diff
@@ -0,0 +1,334 @@
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+
+import { createRoutineCreator } from '../src';
+
+const PREFIX = 'PREFIX';
+const TRIGGER = `${PREFIX}/TRIGGER`;
+const REQUEST = `${PREFIX}/REQUEST`;
+const SUCCESS = `${PREFIX}/SUCCESS`;
+const FAILURE = `${PREFIX}/FAILURE`;
+const FULFILL = `${PREFIX}/FULFILL`;
+const payload = {
+  some: 'data',
+};
+const triggerAction = {
+  type: TRIGGER,
+  payload,
+};
+const requestAction = {
+  type: REQUEST,
+  payload,
+};
+const successAction = {
+  type: SUCCESS,
+  payload,
+};
+const failureAction = {
+  type: FAILURE,
+  payload,
+};
+const fulfillAction = {
+  type: FULFILL,
+  payload,
+};
+const SOME = `${PREFIX}/SOME_LONG_TYPE`;
+const someAction = {
+  type: SOME,
+  payload,
+}
+const OTHER = `${PREFIX}/OTHER`;
+const otherAction = {
+  type: OTHER,
+  payload,
+}
+
+
+describe('createRoutineCreator', () => {
+  it('should be a function', () => {
+    expect(createRoutineCreator).to.be.a('function');
+  });
+
+  it('should create routine with empty stages argument', () => {
+    const createExtendedRoutine = createRoutineCreator();
+    const routine = createExtendedRoutine(PREFIX);
```

Comment:
See no reason to use `createRoutineCreator()` without passing params. Error should be thrown

Reply:
ok